File: dotnet/cafdemalihapa/Directories/Repository/Dotnet/Cafdemalihapa/Scripts/ScriptsDirectory.cs
using cafdemalihapa.Directories.Repository.Dotnet;
using cafdemalihapa.Directories.Repository.Dotnet.Cafdemalihapa;

namespace cafdemalihapa.Directories.Repository.Dotnet.Cafdemalihapa.Scripts
{
    public class ScriptsDirectory
    (
        CafdemalihapaDirectory cafdemalihapaDirectory
    )
    {
        public string GetPath()
        {
            var cafdemalihapaDirectoryPath = cafdemalihapaDirectory.GetPath();
            var scriptsDirectoryPath = Path.Combine(cafdemalihapaDirectoryPath, "scripts");
            return scriptsDirectoryPath;
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Repository/Dotnet/Cafdemalihapa/Scripts/ScriptsDirectory.fs
namespace cafdemwimu.console.Directories.Repository.Dotnet.Cafdemalihapa.Scripts

open System.IO
open cafdemwimu.console.Directories

type ScriptsDirectory(workingDirectory: WorkingDirectory) =
    member _.GetPath() =
        let workingDirectoryPath = workingDirectory.GetPath()
        let scriptsDirectoryPath = Path.Combine(workingDirectoryPath, "scripts")
        scriptsDirectoryPath
